content/en/platform/corda/5.0/deploying-operating/config/tls-certification.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "TLS Certification Installation on REST Worker"
+version: 'Corda 5.0'
+date: '2023-05-29'
+menu:
+  corda5:
+    identifier: corda5-tls-certification-installation
+    parent: corda5-cluster-config
+    weight: 3075
+section_menu: corda5
+---
+
+# TLS Certification Installation on REST Worker
+
+The REST Worker TLS certificate is presented to a client any time a HTTPS connection is made.
+If no specific parameters are provided, the self-signed certificate is used and the connection to the REST Worker is always HTTPS. However, a warning will be emitted into the REST log explaining how to provide parameters for custom TLS certificates.
+There are a two ways that a TLS certificate can be made available to the REST Worker. For either method there are three pieces of information required to install a valid TLS certificate.
+
+* The TLS certificate itself is signed by a Certification Authority (CA) or an intermediary.
+* A private key corresponding to the public key is included into the TLS certificate.
+* The Certification Chain leads up to CA.
+
+## Certificate Information Provided in PEM Format
+
+REST Worker now accepts optional command line arguments: 
+
+* PEM representation of a TLS certificate to be used: `-rtls.crt.path=<path to tls.crt>`
+
+* PEM representation of the private key and of the CA certificate. This could also be a certification path where CA certificate is listed as the last one on the file: `-rtls.key.path=<path to tls.key>`
+
+## Use with Helm Chart
+
+When deploying the C5 cluster to Kubernetes using Helm, certificate information can be provided as a Kubernetes secret. 
```

Comment:
```suggestion
When deploying a Corda cluster to Kubernetes using Helm, certificate information can be provided as a Kubernetes secret. 
```

One of the reasons I thought this might belong in the Deployment section. Seems like something that needs to be considered/performed as part of the deployment process